packages/prisma/seed-app-store.config.json

```diff
@@ -2,300 +2,235 @@
   {
     "/*": "This file is auto-generated and updated by `yarn app-store create/edit`. Don't edit it manually",
     "dirName": "routing-forms",
-    "categories": [
```

Comment:
@zomars I don't know why but prettier keeps changing this depending on the contributor's setup I guess.
I remember reviewing a PR, a day or 2 back which committed this change.